app/public/cantusdata/renderers/csv_renderer.py

```diff
@@ -0,0 +1,20 @@
+from typing import List
+from rest_framework.renderers import BaseRenderer
+
+
+class CSVRenderer(BaseRenderer):
+    """
+    Provides a renderer class which serializes a response to CSV.
+    The renderer assumes that the response data is a list of dictionaries
+    with a consistent set of keys (which become the header row in the csv
+    result).
+    """
+
+    media_type = "text/csv"
```

Comment:
Here, we say that the `CSVRenderer` should be used when the requested media type is "text/csv"...so when our `ManuscriptFolioSetView` is now determining which renderer to use, it will choose the `CSVRenderer` when the request is for "text/csv"